app/components/views/Info/AddToMetaMaskButton/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from "react";
+import { providers } from "ethers";
+import styles from "./index.module.css";
+
+interface Props {
+  address: string;
+  ariaLabel: string;
+  ticker: string;
+  image: string;
+  provider: providers.Web3Provider;
+}
+
+const AddToMetaMaskButton: FC<Props> = (props) => {
+  const { provider } = props.provider;
+  const handleAddToMetaMask = async () => {
+    try {
+      await provider.request?.({
+        method: "wallet_watchAsset",
+        params: {
+          // @ts-ignore complaining that it wants type Any[] for options but heres the interface https://docs.metamask.io/guide/rpc-api.html#wallet-watchasset
+          type: "ERC20",
+          options: {
+            address: props.address,
+            symbol: props.ticker,
+            image: props.image,
+            decimals: 9,
```

Comment:
```suggestion
            decimals: props.decimals,
```
9 for klima, sklima
18 for bct, wsklima, and LP tokens